### ISSUE TITLE
Continuation of apprenticeship post 4 weeks?

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/apprenticeship#7]
In the FAQ the apprenticeship is listed as "initially 4 weeks". 

What is the process for deciding how things will continue after those initial 4 weeks? 

I am teaching at Founders and Coders for 3 weeks after finishing the apprenticeship, but if there is not a possibility to continue with dwyl after I am finished with that then I need to pursue other employment

I'm enthusiastic about continuing with dwyl if that is an option available to me, but it would be good to know if it is, and in the case that it is an option, what the specifics of that continuation would be.
e.g. day rate, what the work would be, when it would be, where I would be expected to work from etc.

**Comment by nelsonic** [comment]
@ZooeyMiller _great_ question, the first step is for you and @finnhodgkin to give us a _lot_ of feedback on what you would _like_ to learn & do.

**Comment by ghost** [comment]
I can half-answer a few of these now:

day rate - this will be assessed following the apprenticeship, depending on skill level likely somewhere between £75/day-£120/day
what the work would be - could be working on client projects or internal projects such as the dwyl chat app - there is some flexibility here hence Nelson's question above
when it would be - this would depend on the kind of work you wanted to pursue (internal / external)
where I would be expected to work from etc. - dwyl is a remote-first company, so you can work from wherever you wish, although you might be needed at No. 16 sometimes for meetings, events etc

**Comment by ZooeyMiller** [comment]
What I would really like to do in terms of technology is get more acquainted with Elixir, learn Phoenix and work on some more back end technologies. 

I like front end but I find working on front end without a set design in place quite difficult, when given a design I find front end and styling quite enjoyable and can be relatively productive, but when design is mixed in I hit a bit of a brick wall.

I'm very interested in increasing performance in a way that is meaningful to users, and would love to learn more about it.

Something I have been considering in my spare time is to start making Elm tutorial videos. I really like the videos on Front end masters but I'd like to make some free videos on youtube walking through some Elm basics. If that was something I could do with dwyl that would be amazing. 

In terms of projects I am quite easy with what I work on, I would prefer to work on projects based on helping people, especially if those projects can be for not for profits or charities.

I would like to work on projects which have a clear structure and defined users that we can make decisions based on, with a product owner I can communicate with. 

I'd mostly like to work from the Palmer's Road offices, but it would be good to work remotely for a week at some points to visit family. 

Hope that gives some helpful info :smile:

**Comment by finnhodgkin** [comment]
@nelsonic It's quite hard to follow @ZooeyMiller's feedback because my sentiments are practically identical and also v. similar to my [application](https://github.com/dwyl/hq/issues/356#issuecomment-307851334).

I'm loving the Elm work! Right now seems to be the perfect time for it—there's a super friendly, growing community _and_ it's young enough for there to be a lot of room for new, useful packages. On the other hand I don't want to be pigeonholed into pure front-end land forever (Elixir and Phoenix are calling me :drooling_face:).  So yeah, Elm is fun but variety is the bomb.

In terms of client vs internal I'm pretty easy. Working on a bunch of unexpected, charitable client work is fantastic but—at least in my past—the indecisiveness can get frustrating ('wtf the exact thing I asked for isn't useful'). Again, variety +1

On a side note I'm a little confused by the question because I'm _so unused_ to being asked what I _want_ to do in relation to work. Are you asking because there's room at dwyl for me to decide my own fate? I love coding and helping others to do the same, and so far dwyl has been perfect for this (keep me :pray::wink:), but it's hard to tell whether I'm an asset or a temporary distraction. I can see that from your side feedback is super useful, but it's unclear whether it's feedback to decide where we should be within dwyl or whether we should be there at all..?

I can't wait to get back to work next week. Hope you're having a lovely time over the sea :smile:

**Comment by ghost** [comment]
Although the original apprenticeship has finished, the work has continued after the four weeks with finn and zooey continuing to learn and contribute to the learn elixir repo
